src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -451,6 +452,14 @@ class RsErrorAnnotator : RsAnnotatorBase(), HighlightRangeExtension {
 
     private fun checkAttr(holder: AnnotationHolder, attr: RsAttr) {
         checkImplBothCopyAndDrop(holder, attr)
+        checkAttrStableUsed(holder, attr)
+    }
+
+    // E0497: Stable attribute can only be used in the standard library
+    private fun checkAttrStableUsed(holder: AnnotationHolder, attr: RsAttr) {
+        if (attr.metaItem.name == "stable" && attr.cargoProject?.doesLookLikeRustLangRust() != true) {
```

Comment:
I think use can just look at the origin of the `attr`'s package:
```kotlin
if (attr.metaItem.name == "stable" && attr.containingCargoPackage?.origin != PackageOrigin.STDLIB) {
```
So you can drop https://github.com/intellij-rust/intellij-rust/pull/3806 dependency. @vlad20012 Am I right?

Reply:
@mchernyavsky Looks like if a project is rustc itself, it will produce false positive error annotation because in this case, `stdlib` crates are part of the project workspace